deployment/terraform/create.go

```diff
@@ -426,16 +427,41 @@ func (t *Terraform) setupProxyServer(extAgent *ssh.ExtAgent) {
 			backends += "server " + addr.PrivateIP + ":8065 max_fails=0;\n"
 		}
 
+		cacheObjects := "10m"
+		cacheSize := "3g"
+		// Extracting the instance class from the type.
+		// Usually they are of the form (m7/c7).(large/2xlarge/4xlarge/..)
+		parts := strings.Split(t.config.ProxyInstanceType, ".")
+		if len(parts) > 1 {
+			// Hack alert. There can be other higher variants like 12xlarge or even metal. But we are keeping it simple for now.
```

Comment:
I'd rather we failed, logged an error or something if we find anything we don't understand/expect. Mostly to avoid subtle bugs as we'll forget about this eventually.

Reply:
You mean you want to limit to only 2x/4x/8x instances? That sounds a bit overly restrictive.. :thinking:

Reply:
Yes indeed, I was thinking of doing something like this. I think after your ES PR goes in, which starts to use the aws client, I can easily do this as an OSF project.

Reply:
Great idea!

Reply:
Neat, thanks! :)